packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -552,7 +552,7 @@ export async function collectFilesForIndex(
     }
 
     const isBuildOrBin = (filePath: string) => {
-        const k = /[/\\](bin|build|node_modules|env|\.idea)[/\\]/i
+        const k = /[/\\](bin|build|node_modules|env|\.idea|\.venv|venv)[/\\]/i
```

Comment:
@leigaol why don't we use vscode's `'files.exclude'` and/or `'search.exclude'` settings? That has these benefits:

1. allows customer to control the excludes
2. benefits from vscode default exclude patterns (which include what you have here plus more)